Purchases/Networking/Caching/CustomerInfoCallback.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfoCallback.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+struct CustomerInfoCallback: CacheKeyProviding, CustomerInfoCompletion {
```

Comment:
will review on, but initially i'm confused why CustomerInfoCallback implements the completion...other callbacks don't seem to follow this pattern

Reply:
This is me forgetting to clean up a previous refactor 🤦‍♂️ sorry!

Reply:
oh yay then i understand it!